radar.yml

```diff
@@ -19,4 +19,12 @@ management_portal_config:
     subject_endpoint: api/subjects/
     source_type_endpoint: api/source-types/
     source_data_endpoint: api/source-data/
-    source_endpoint: api/sources/
\ No newline at end of file
+    source_endpoint: api/sources/
+
+# Timeout duration for every source-type to decide source status whether its connected or not.
+# A source-type should be defined by following the convention of producer_model as mentioned in
+# the specification in radar-schemas
+# timeout should be specified as {@code PnDTnHnMn.nS}.
+source-type-connection-timeout:
+    android_phone: 2H
```

Comment:
Shouldn't the default be `PT2H` then? Please mention the ISO 6801 spec.

Reply:
Indeed. My bad